tests/test_biteme.py

```diff
@@ -0,0 +1,31 @@
+import pathlib
+
+import biteme
+
+
+class TestBiteInfo:
+    def test___init__(self) -> None:
+        bite_info = biteme.BiteInfo(
+            number=1,
+            title="title",
+            description="description",
+            level="level",
+            tags=["tag1", "tag2"],
+            free=True,
+            score=2,
+            function="function",
+        )
+        assert bite_info.number == 1
+        assert bite_info.title == "title"
+        assert bite_info.description == "description"
+        assert bite_info.level == "level"
+        assert bite_info.tags == frozenset({"tag1", "tag2"})
+        assert bite_info.free is True
+        assert bite_info.score == 2
+        assert bite_info.function == "function"
+
+
+def test_download_bite(api_key: str, tmp_path: pathlib.Path) -> None:
+    bite_dir = biteme.download_bite(api_key, 1, tmp_path)
```

Comment:
~~do we want to cleanup the generated directory when done?~~ ignore, `tmp_path` should take care of that

Reply:
should we also make an assertion about the extracted content (should be static)?

Reply:
I want to mock out the request so when I do that, I'll add assertions about the content because the content will be dictated by some test fixture.

Reply:
Unit vs integration test, I suppose. Probably want both!

Reply:
Yep can do and break them out into different files or use pytest markers. Also for unittest are you going with `patch` or use something like `vcr` (cassettes)? Can be a later version though right?

Reply:
Yeah, will do this in a later version.